Library/Formula/tippecanoe.rb

```diff
@@ -23,8 +23,10 @@ def install
     path.write <<-EOS.undent
       {"type":"Feature","properties":{},"geometry":{"type":"Point","coordinates":[0,0]}}
     EOS
-    output = `#{bin}/tippecanoe -o test.mbtiles #{path}`.strip
+
+    `#{bin}/tippecanoe -o test.mbtiles #{path}`
```

Comment:
@DomT4 Thanks for the comments.

The output was returning an empty string, instead of the expected stdout message. 

I made a call that the stdout-based test was a tad brittle, as it didn't tell us anything extra than the exit code already did (the tool works). As such, rather than restore the original test, I switched to a direct check that the expected file output exists.

Nonetheless, if you reckon I should spend more time on reviving the original test, I can dig into it some more.